### ISSUE TITLE
`parse': undefined method `size' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by rays** [zerowidth/camper_van#7]
I get the following error when attempting to connect, after this error the app stops running

/Library/Ruby/Gems/1.8/gems/camper_van-0.0.2/lib/camper_van/command_parser.rb:32:in `parse': undefined method`size' for nil:NilClass (NoMethodError)
    from /Library/Ruby/Gems/1.8/gems/camper_van-0.0.2/lib/camper_van/ircd.rb:66:in `receive_line'
    from /Library/Ruby/Gems/1.8/gems/camper_van-0.0.2/lib/camper_van/server.rb:97:in`receive_line'
    from /Library/Ruby/Gems/1.8/gems/eventmachine-0.12.10/lib/em/protocols/linetext2.rb:64:in `receive_data'
    from /Library/Ruby/Gems/1.8/gems/eventmachine-0.12.10/lib/eventmachine.rb:256:in`run_machine'
    from /Library/Ruby/Gems/1.8/gems/eventmachine-0.12.10/lib/eventmachine.rb:256:in `run'
    from /Library/Ruby/Gems/1.8/gems/camper_van-0.0.2/lib/camper_van/server.rb:21:in`run'
    from /Library/Ruby/Gems/1.8/gems/camper_van-0.0.2/bin/camper_van:57
    from /usr/bin/camper_van:19:in `load'
    from /usr/bin/camper_van:19

**Comment by rays** [comment]
Wrong version of Ruby...

**Comment by zerowidth** [comment]
Ah, that was easy. Sorry I didn't get back to you earlier. What version of ruby were you using, and what are you using now?

**Comment by tru** [comment]
I am interested to know that as well, I have exactly the same problem. It seems to choke on the PASS line, which is the first that is sent from irssi:

```
2012-04-25 13:45:28   855  INFO CamperVan : starting server on 127.0.0.1:6667
2012-04-25 13:45:35   855  INFO CamperVan : got connection from 127.0.0.1
2012-04-25 13:45:35   855 DEBUG CamperVan : irc -> PASS plexapp:<HIDDEN>
2012-04-25 13:45:35   855  INFO CamperVan : closed connection from 127.0.0.1
/var/lib/gems/1.8/gems/camper_van-0.0.3/lib/camper_van/command_parser.rb:33:in `parse': undefined method `size' for nil:NilClass (NoMethodError)
    from /var/lib/gems/1.8/gems/camper_van-0.0.3/lib/camper_van/ircd.rb:66:in `receive_line'
    from /var/lib/gems/1.8/gems/camper_van-0.0.3/lib/camper_van/server.rb:97:in `receive_line'
    from /var/lib/gems/1.8/gems/eventmachine-0.12.10/lib/em/protocols/linetext2.rb:64:in `receive_data'
    from /var/lib/gems/1.8/gems/eventmachine-0.12.10/lib/eventmachine.rb:256:in `run_machine'
    from /var/lib/gems/1.8/gems/eventmachine-0.12.10/lib/eventmachine.rb:256:in `run'
    from /var/lib/gems/1.8/gems/camper_van-0.0.3/lib/camper_van/server.rb:21:in `run'
    from /var/lib/gems/1.8/gems/camper_van-0.0.3/bin/camper_van:64
    from /usr/local/bin/camper_van:19:in `load'
    from /usr/local/bin/camper_van:19
```
